11LessonEleven/homework/css/inspired.css

```diff
@@ -0,0 +1,39 @@
+.top {
+    background-image: url("../img/bg.jpg");
```

Comment:
in this case make sure that when you add an image as a background it does not repeat